From 9c79ec5f65dbb8ebadb56c4239c3d9bad6a567c4 Mon Sep 17 00:00:00 2001
From: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Date: Tue, 23 Feb 2016 10:24:44 +0530
Subject: [PATCH 665/793] atg: misc: use memset_io

No functional change instead of using a loop move to memset_io.

Signed-off-by: Shubhrajyoti Datta <shubhraj@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/misc/xilinx_trafgen.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/drivers/misc/xilinx_trafgen.c b/drivers/misc/xilinx_trafgen.c
index 5528de8c..421495d 100644
--- a/drivers/misc/xilinx_trafgen.c
+++ b/drivers/misc/xilinx_trafgen.c
@@ -372,8 +372,7 @@ static void xtg_access_rams(struct xtg_dev_info *tg, int where,
 
 	switch (flags) {
 	case XTG_WRITE_RAM_ZERO:
-		for (index = 0; count > 0; index++, count -= 4)
-			writel(0x0, tg->regs + where + index * 4);
+		memset_io(tg->regs + where, 0, count);
 		break;
 	case XTG_WRITE_RAM:
 		for (index = 0; count > 0; index++, count -= 4)
-- 
1.7.5.4

